frontend/src/store/Books/BooksReducer.ts

```diff
@@ -25,21 +25,19 @@ const booksReducer = (
 	switch (action.type) {
```

Comment:
Must return output to see where went wrong, this part causes the allBookState to return never. Since it doesn't return the exact things that we wanted. 

const booksReducer = (
	state: BooksState = initialState,
	action: DispatchBookActions
):**BooksState** => {}